rafay/resource_breakglassaccess.go

```diff
@@ -272,7 +272,7 @@ func expandGroupExpiry(p []interface{}) []*systempb.GroupExpiryDetails {
 		ge := &systempb.GroupExpiryDetails{}
 
 		if v, ok := expiryMap["expiry"].(int); ok {
-			ge.Expiry = float32(v)
+			ge.Expiry = float64(v)
```

Comment:
@venkatm-rafay Is this required?

Reply:
yes @vasu-rafay it's throwing the error while doing make build. The field is of type Float64.